<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Clip</class>
 <widget class="QWidget" name="Clip">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>32</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0,0,0,0,0,0">
   <property name="spacing">
    <number>5</number>
   </property>
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>开始</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTimeEdit" name="timeEditStart">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="currentSection">
      <enum>QDateTimeEdit::HourSection</enum>
     </property>
     <property name="displayFormat">
      <string>hh:mm:ss:zzz</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>结束</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTimeEdit" name="timeEditEnd">
     <property name="currentSection">
      <enum>QDateTimeEdit::HourSection</enum>
     </property>
     <property name="displayFormat">
      <string>hh:mm:ss:zzz</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>时长</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTimeEdit" name="timeEditDuration">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="currentSection">
      <enum>QDateTimeEdit::HourSection</enum>
     </property>
     <property name="displayFormat">
      <string>hh:mm:ss:zzz</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButtonCut">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="res.qrc">
       <normaloff>:/cut.png</normaloff>:/cut.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar">
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
